### ISSUE TITLE
Provide Server Hostname and Serial Number properties for Contract Warranty status output from Get-HPOVRemoteSupportEntitlementStatus.

### ISSUE THREAD

**Issue by ChrisLynchHPE** [HewlettPackard/POSH-HPOneView#290]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
When executing `Get-HPOVRemoteSupportEntitlementStatus` and within the `HPOneView.RemoteSupport.ContractAndWarrantyStatus` class object, add `ResourceSerialNumber` and display `serverHostname` for Server Hardware in the `ResourceName` field.

```
ResourceName ResourceSerialNumber IsEntitled EntitlementPackage EntitlementStatus CoverageDays ObligationStartDate ObligationEndDate
------------ -------------------- ---------- ------------------ ----------------- ------------ ------------------- -----------------
My OS Name   AB1234566            True                                                         1/1/1970 00:00:00   1/1/1970 00:00:00
```

### Actual Behavior
None.

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`): `3.10.1416.3042`
HPE OneView Appliance Version:  `3.10.00.4`

**Comment by ChrisLynchHPE** [comment]
The enhancement will include:

1. The `ContractAndWarrantyStatus` Class Object will be updated to include a `ResourceSerialNumber` property. 
1. Update .stor to accept `ResourceSerialNumber` property.
1. The Format View for `HPOneView.RemoteSupport.ContractAndWarrantyStatus` will change to no longer default display `ObligationStartDate` in order to make room to display `ResourceSerialNumber`.
1. `Get-HPOVRemoteSupportEntitlementStatus` will now handle the input object type based on category type, and display `serverName` property value for `server-hardware` resources.

**Comment by ChrisLynchHPE** [comment]
This enhancement will be released in the next 3.10 library update.

**Comment by ChrisLynchHPE** [comment]
This is now added in [Release 3.10.1471.1581](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.10.1471.1581), which has also been published to the [PowerShellGallery](https://www.powershellgallery.com/packages/HPOneView.310).

Closing.
